src/react-native/expo-env-file.ts

```diff
@@ -0,0 +1,53 @@
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import * as clack from '@clack/prompts';
+import chalk from 'chalk';
+import fs from 'fs';
+import * as Sentry from '@sentry/node';
+import { RNCliSetupConfigContent } from './react-native-wizard';
+import { addToGitignore } from './git';
+
+const EXPO_ENV_LOCAL_FILE = '.env.local';
+
+export async function addExpoEnvLocal(
+  options: RNCliSetupConfigContent,
+): Promise<boolean> {
+  const newContent = `#DO NOT COMMIT THIS\nSENTRY_AUTH_TOKEN=${options.authToken}\n`;
+
+  const added = await addToGitignore(EXPO_ENV_LOCAL_FILE);
+  if (added) {
+    clack.log.success(
+      `Added ${chalk.cyan(EXPO_ENV_LOCAL_FILE)} to .gitignore.`,
+    );
+  } else {
+    clack.log.error(
+      `Could not add ${EXPO_ENV_LOCAL_FILE} to .gitignore, please add it to not commit your auth key.`,
```

Comment:
l (feel free to ignore): we might wanna make the filename `chalk.cyan`

Reply:
Make sense, also the sucess message has it, thanks.